storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1023,6 +1025,15 @@ struct my_option xb_client_options[] =
     "privileges before performing any query.", &opt_check_privileges,
    &opt_check_privileges, 0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+  {"read_buffer_size",
+   OPT_XTRA_READ_BUFFER_SIZE,
+   "Set datafile read buffer size, given value is scaled up to page size."
+   " Default is 10Mb.",
+   &opt_read_buffer_size,
+   &opt_read_buffer_size,
+   0, GET_UINT, OPT_ARG, UNIV_PAGE_SIZE_MAX*640,
```

Comment:
UNIV_PAGE_SIZE_MAX is different between 2.3 and 2.4. Please make sure to take it to attention.